### PR TITLE
VDS-2780: Implement frontend part of Reset password scenario

### PR DESCRIPTION
**Pull request by Pav-el**
Description: 

--

QA-test: VDS-2152

Demo-test: VDS-2153

Download artifact URL: ghcr.io/virtocommerce/demo-xapi-app:1.0.0-pr-67-vds-2780-da01591f

### CODE REVIEW

File: src/app/modules/reset-password/email-request/email-request.component.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            email
+            required />
+          <span *ngIf="form.invalid && form.dirty" class="d-block text-danger">Enter a valid email address</span>
+          <button [disabled]="form.invalid" class="btn btn-primary mt-5 flex-fill flex-sm-grow-0" type="submit">
+              <span class="mx-5">Submit</span>
+          </button>
+        </form>
+      </div>
+      <aside class="d-none d-md-block col offset-2">
+        <img class="img-fluid user-select-none" src="assets/images/login.svg" alt="" />
```
Comment (ilyawzrd) [inline src/app/modules/reset-password/email-request/email-request.component.html]: Write something to the 'alt' attribute

File: src/app/modules/reset-password/reset-password.component.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
```
Comment (ilyawzrd) [inline src/app/modules/reset-password/reset-password.component.ts]: Can we remove this component? It looks unused

File: src/app/modules/reset-password/reset-password.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (ilyawzrd) [inline src/app/modules/reset-password/reset-password.component.spec.ts]: Same as reset-password.component.ts

File: src/app/modules/reset-password/reset-password.component.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<p>reset-password works!</p>
```
Comment (ilyawzrd) [inline src/app/modules/reset-password/reset-password.component.html]: Same as reset-password.component.ts

**Review by ilyawzrd — CHANGES_REQUESTED** [review]

**Review by ilyawzrd — APPROVED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**